### PR TITLE
Change breadcrumbs props

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Change the props passed to the Breadcrumbs and add it to the SearchResult page.

#### What problem is this solving?
Now the breadcrumbs component can be used on the SearchResult page.

#### How should this be manually tested?
[workspace](https://felipeoficial--storecomponents.myvtex.com/eletronicos/computadores/c)

#### Linked to

[breadcrumbs](https://github.com/vtex-apps/breadcrumb/pull/9)

#### Screenshots or example usage
image

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [x] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/ProductSearchContextProvider.js [hunk 1]
```diff
@@ -17,6 +17,33 @@ class ProductSearchContextProvider extends Component {
     loading: true,
   }
 
+  getBreadcrumbsProps() {
+    const { params } = this.props
```
Comment (ClaudivanFilho) [inline react/ProductSearchContextProvider.js]: Coloca assim, melhor :)
```
const { params : { caegory, department, term } } = this.props
```

Comment (salesfelipe) [reply react/ProductSearchContextProvider.js]: 👍

File: react/ProductContextProvider.js [hunk 2]
```diff
@@ -119,6 +119,11 @@ class ProductContextProvider extends Component {
       product,
     }
 
+    const breadCrumbsProps = {
```
Comment (brunojdo) [inline react/ProductContextProvider.js]: add a doc for this decision, and `breadcrumbsProps` is better

Comment (salesfelipe) [reply react/ProductContextProvider.js]: 👍

**Pull request merged**